mlfinlab/tests/test_risk_estimators.py

```diff
@@ -257,6 +257,8 @@ def test_detoned():
                          [0.1, 1, -0.3],
                          [-0.1, -0.3, 1]])
 
+        e_val, e_vec = np.linalg.eigh(corr)
```

Comment:
Can we change the naming to "eigenvalues" and "eigenvectors" just to keep it more readable.

Reply:
Done. Turned out this part is unused in the current version of de-toning 😄